Program2.java

```diff
@@ -0,0 +1,58 @@
+import java.util.*;
+
+
+public class Program2 {
+
+	 
+    public static void main(String[] args) {
+       
+    	Scanner in = new Scanner (System.in); 
+        System.out.print("Input 1-7 as a day of week: ");
+        
+        
+        if (in.hasNextInt())
+        {        	       	
+        	int num = in.nextInt();
```

Comment:
nextInt() method will throw an exception if int value cannot be get from the input so no need to check if Int available